doc/tutorials/acrn_on_qemu.rst

```diff
@@ -0,0 +1,289 @@
+   This can be verified from ``dmesg`` as shown below,
+
+   .. code-block:: none
+
+      guestl1@ACRNSOS:~$ dmesg | grep ACRN
+      [    0.000000] Hypervisor detected: ACRN
+      [    2.337176] ACRNTrace: Initialized acrn trace module with 4 cpu
+      [    2.368358] ACRN HVLog: Initialized hvlog module with 4 cpu
+      [    2.727905] systemd[1]: Set hostname to <ACRNSOS>.
+
+8. When shutting down, make sure to cleanly destroy the Service VM to prevent crashes in subsequent boots. This can be done using,
+
+      .. code-block:: none
+
+         virsh destory ACRNSOS # where ACRNSOS is the virsh domain name.
+
+
+Service VM Networking updates for User VM
+*****************************************
+If you are planning to enable networking for User VM(L2 guest), the following steps might be helpful.
+
+1. Edit your :file:`/etc/netplan/01-netcfg.yaml` file to add acrn-br0 as below,
+
+   .. code-block:: none
+
+      network:
+         version: 2
+         renderer: networkd
+         ethernets:
+            enp1s0:
+               dhcp4: no
+         bridges:
+            acrn-br0:
+               interfaces: [enp1s0]
+               dhcp4: true
+               dhcp6: no
+
+2. Apply the new network configuration by,
+
+      .. code-block:: none
+
+         $ cd /etc/netplan
+         $ sudo netplan generate
+         $ sudo netplan apply
+
+3. Create a tap interface (tap0) and add the tap interface as part of the acrn-br0.
+   Copy the following files to :file:`/usr/lib/system/network`,
+
+   - `50-acrn.network <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/misc/acrnbridge/acrn.network>`__
+   - `50-tap0.netdev <https://raw.githubusercontent.com/projectacrn/acrn-hypervisor/master/misc/acrnbridge/tap0.netdev>`__
+
+4. Restart ACRNSOS guest (L1 guest) to complete the setup and start with bring-up of User VM
+
+
+Bring-up User VM (L2 Guest)
+***************************
+1. Build the device-model, using make devicemodel and copy acrn-dm to ACRNSOS guest (L1 guest) directory ``/usr/bin/acrn-dm``
+
+   .. note::
+      It should be already built as part of .._Install ACRN Hypervisor.
+
+2. On the ACRNSOS guest, install shared libraries for acrn-dm (if not already installed).
+
+   .. code-block:: none
+
+      sudo apt-get install libpciaccess-dev
+
+3. Install latest `IASL tool <https://acpica.org/downloads>`_ and copy the binary to ``/usr/sbin/iasl``.
+   For this setup, used IASL 20200326 version but anything after 20190215 should be good.
+
+4. Build ACRN User VM Kernel by,
+
+      a. Please clone the latest stable version or master as show below,
+
+         .. code-block:: none
+
+            $ git clone https://github.com/projectacrn/acrn-kernel
+            $ cd acrn-kernel
+
+      b. Copy kernel_config_uos to .config
+      c. make
+      d. Copy ``arch/x86/boot/bzImage`` to the ACRNSOS (L1 guest). Need this to launch the User VM (L2 guest)
```

Comment:
I'm not 100% clear what this means, can you clarify?

(Note: I see later you use that file from `/home/guestl1/acrn-dm-bins/bzImage_uos` which is on the Service Vm rootfs. Is that what we should instruct the user to do here?

Reply:
Reworded, please check if this clarifies.